docs/usage/docker.md

```diff
@@ -265,6 +270,46 @@ Renovate can authenticate with AWS ECR using AWS access key id & secret as the u
 }
 ```
 
+##### Using the login password (basic auth)
+
+To authenticate using basic auth, simply provide the password in your config. For example:
+
+```json
+{
+  "hostRules": [
+    {
+      "hostType": "docker",
+      "matchHost": "12345612312.dkr.ecr.us-east-1.amazonaws.com",
+      "encrypted": {
+        "password": "w...A"
+      }
+    }
+  ]
+}
+```
+
+To obtain a login password, using the command `aws ecr get-login-password` or the corresponding API.
```

Comment:
```suggestion
To get a login password, use the command `aws ecr get-login-password` or the corresponding API.
```

Do we need to link to the API docs for this? Or do we expect AWS users to know this already?

Reply:
I would generally assume that anyone using Codeartifact would know about the AWS credentials for CLI or API use.  We can certainly link to the docs if it helps with clarity:
https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html